backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeNodeToolOnDiagramEventHandler.java

```diff
@@ -106,7 +106,8 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
                     .map(CreateNodeTool.class::cast);
             // @formatter:on
             if (optionalTool.isPresent()) {
-                IStatus status = this.executeTool(editingContext, diagramContext, input.getDiagramElementId(), optionalTool.get(), input.getStartingPositionX(), input.getStartingPositionY(),
+                Optional<UUID> optionalDiagramElementId = Optional.ofNullable(input.getDiagramElementId());
```

Comment:
This file should not be *that* impacted by the change from UUID to String

Reply:
Done